src/main/java/org/broadinstitute/hellbender/transformers/PalindromeArtifactClipReadTransformer.java

```diff
@@ -0,0 +1,114 @@
+package org.broadinstitute.hellbender.transformers;
```

Comment:
Is there a way you can test this?  An automated test of some kind?  We have utilities that can generate fake reads....

Reply:
done